core/IO.carp

```diff
@@ -1,5 +1,7 @@
 (system-include "carp_io.h")
 
+(load-once "Platform.carp")
```

Comment:
I don't think this should be necessary.

(it makes the `Core.carp` file harder to reason about)

Reply:
That’s right, this is left over from the previous commit. Sorry about that!